### ISSUE TITLE
bug 0.9.4 CSS broken: button with caret moved to next line

### ISSUE THREAD

**Issue by iyel** [angular-ui/ui-select#494]

**Comment by Morgul** [comment]
I fixed this in a different way from how it was suggested; I used [calc](http://caniuse.com/#feat=calc). My solution's very specific for my use case, but it basically boils down to this:

```css
.ui-select-match > button.col-sm-10 {
    width: calc(100% - 34px);
}

.ui-select-match > button.col-sm-8 {
    width: calc(100% - 73px);
}
```

This means that the caret button will need to be a fixed width (this is easily doable in my experience), and you will need to apply a class to this button if clear selection is enabled and something is selected. That class will need to override the calc width, but that's also easily doable.

While I'm sure that using something like flex-box might be more desirable, I've found the `calc()` is pretty simple and very well supported. It's how I'd suggest solving this issue, personally.

**Comment by kashesandr** [comment]
Faced with the same issue.

**Comment by brianfeister** [comment]
Fix released in v0.9.6
